src/MSBuildLocator/VisualStudioInstanceQueryOptions.cs

```diff
@@ -33,5 +33,11 @@ public class VisualStudioInstanceQueryOptions
         ///     Working directory to use when querying for instances. Ensure it is the project directory to pick up the right global.json.
         /// </summary>
         public string WorkingDirectory { get; set; } = Environment.CurrentDirectory;
+
+        /// <summary>
+        ///     This variable enables the removal of the existing restriction on querying installed Visual Studio instances 
+        ///     with a runtime version lower than or equal to the one in the current environment.
+        /// </summary>
+        public bool AllowQueryAllRuntimeVersions { get; set; } = Environment.GetEnvironmentVariable("DOTNET_MSBUILD_QUERY_ALL_RUNTIMES") == "1";
```

Comment:
@baronfel is there a good scenario for the inverse--.NET Framework locator finding SDK installations?

Reply:
This would be part of the bucket of scenarios where the user is almost certainly not going to be able to _load_ the MSBuild, but they might just want to list/shell-invoke/etc the MSBuild? It could be useful, but a) wouldn't be as high of a priority in my mind, and b) would need to be behind whatever 'I promise I know this won't work if I try to in-proc this' flag/gate we've set up here.